backport_packages/refactor_backport_packages.py

```diff
@@ -0,0 +1,786 @@
+
+        .. code-block:: diff
+
+            --- ./airflow/providers/google/cloud/example_dags/example_datacatalog.py
+            +++ ./airflow/providers/google/cloud/example_dags/example_datacatalog.py
+            @@ -37,7 +37,7 @@
+                 CloudDataCatalogUpdateTagTemplateOperator,
+             )
+             from airflow.utils.dates import days_ago
+            -from airflow.utils.helpers import chain
+            +from airflow.providers.odbc.utils.helpers import chain
+
+             default_args = {"start_date": days_ago(1)}
+
+
+        """
+        # noinspection PyUnusedLocal
+        def odbc_package_filter(node: LN, capture: Capture, filename: Filename) -> bool:
+            return filename.startswith("./airflow/providers/odbc/")
+
+        os.makedirs(os.path.join(get_target_providers_folder(), "odbc", "utils"), exist_ok=True)
+        copyfile(
+            os.path.join(get_source_airflow_folder(), "airflow", "utils", "__init__.py"),
+            os.path.join(get_target_providers_package_folder("odbc"), "utils", "__init__.py")
+        )
+        copy_helper_py_file(os.path.join(
+            get_target_providers_package_folder("odbc"), "utils", "helpers.py"))
+
+        (
+            self.qry.
+            select_module("airflow.utils.helpers").
+            filter(callback=odbc_package_filter).
+            rename("airflow.providers.odbc.utils.helpers")
+        )
+
+    def refactor_papermill_package(self):
+        """
+        Fixes to "papermill" providers package.
+
+        Copies some of the classes used from core Airflow to "common.utils" package of the
+        the provider and renames imports to use them from there.
+
+        We copy lineage.py and it's __init__.py and we change import as in example diff:
+
+        .. code-block:: diff
+
+            --- ./airflow/providers/papermill/example_dags/example_papermill.py
+            +++ ./airflow/providers/papermill/example_dags/example_papermill.py
+            @@ -26,8 +26,8 @@
+             import scrapbook as sb
+
+             from airflow import DAG
+            -from airflow.lineage import AUTO
+            -from airflow.operators.python import PythonOperator
+            +from airflow.providers.papermill.utils.lineage import AUTO
+            +from airflow.operators.python_operator import PythonOperator
+             from airflow.providers.papermill.operators.papermill import PapermillOperator
+             from airflow.utils.dates import days_ago
+             from airflow.version import version
+
+
+        Note also that copied lineage __init__.py needs to be refactored as well because it uses
+        Operator class (which is not existing in Airflow 1.10.*. We have a base operator template
+        prepared that imports the BaseOperator as an Operator and copy it as "base.py" in the
+        papermill.utils package (from template_base_operator.py) and we rename import to use it from there:
```

Comment:
This seems like a lot of work, and how well have we tested it actually works?

Also given how small Papermill operator is and the problems it's causing/special cases it needs: should we not do a backport release of it.

Reply:
Yes. I will exclude it. It is in fact very problematic and won't work with lineage at all anyway,